tracer/missing-nullability-files.csv

```diff
@@ -36,6 +36,7 @@ src/Datadog.Trace/SpanExtensions.Core.cs
 src/Datadog.Trace/SpanExtensions.cs
 src/Datadog.Trace/SpanExtensions.Framework.cs
 src/Datadog.Trace/SpanKinds.cs
+src/Datadog.Trace/SpanLink.cs
```

Comment:
We should add `#nullable enabled` (and related nullability annotations) to the new `SpanLink.cs` file to avoid adding more file to this list. We're trying to get it down to zero someday 😅